frontend/src/AlertConfiguration/AlertConfigurationComponents/ConfigList.js

```diff
@@ -1,49 +1,24 @@
-import React, { useState, Fragment } from 'react';
-import AppBar from '@material-ui/core/AppBar';
-import CssBaseline from '@material-ui/core/CssBaseline';
+import React from 'react';
+import Container from '@material-ui/core/Container';
 import { makeStyles } from '@material-ui/core/styles';
-import Toolbar from '@material-ui/core/Toolbar';
-import Typography from '@material-ui/core/Typography';
-import ControlPanel from './ControlPanel';
-import Footer from './Footer';
-import ConfigList from './ConfigList';
+import ConfigCard from './ConfigCard';
 
-const useStyles = makeStyles((theme) => ({
-  main: {
-    display: 'flex',
-    flexDirection: 'column',
-    minHeight: '100vh',
-  },
-}));
+const useStyles = makeStyles({
+  configList: {
+    marginTopBottom: '50',
+  }
+});
 
-export default function AlertConfiguration() {
+export default function ConfigList({ configs }) {
```

Comment:
nit: since there are many components for the alert configuration, could we also add some small one/two-line description for each functional component on what the component does/what the inputs are.

so not just the `ConfigList` component but the others too :)

Reply:
Completed